src/test/java/com/conveyal/gtfs/graphql/GTFSGraphQLTest.java

```diff
@@ -0,0 +1,229 @@
+    }
+
+    // tests that the trips of a feed can be fetched
+    @Test
+    public void canFetchTrips() throws IOException {
+        assertThat(queryGraphQL("feedTrips.txt"), matchesSnapshot());
+    }
+
+    // TODO: make tests for schedule_exceptions / calendar_dates
+
+    // tests that the stop times of a feed can be fetched
+    @Test
+    public void canFetchStopTimes() throws IOException {
+        assertThat(queryGraphQL("feedStopTimes.txt"), matchesSnapshot());
+    }
+
+    // tests that the stop times of a feed can be fetched
+    @Test
+    public void canFetchServices() throws IOException {
+        assertThat(queryGraphQL("feedServices.txt"), matchesSnapshot());
+    }
+
+    /**
+     * attempt to fetch more than one record with SQL injection as inputs
+     * the graphql library should properly escape the string and return 0 results for stops
+     */
+    @Test
+    public void canSanitizeSQLInjectionSentAsInput() throws IOException {
+        Map<String, Object> variables = new HashMap<String, Object>();
+        variables.put("namespace", testInjectionNamespace);
+        variables.put("stop_id", Arrays.asList("' OR 1=1;"));
+        assertThat(
+            queryGraphQL(
+                "feedStopsByStopId.txt",
+                variables,
+                testInjectionDataSource
+            ),
+            matchesSnapshot()
+        );
+    }
+
+    /**
+     * attempt run a graphql query when one of the pieces of data contains a SQL injection
+     * the graphql library should properly escape the string and complete the queries
+     */
+    @Test
+    public void canSanitizeSQLInjectionSentAsKeyValue() throws IOException, SQLException {
+        // manually update the route_id key in routes and patterns
+        String injection = "'' OR 1=1; Select ''99";
+        Connection connection = testInjectionDataSource.getConnection();
+        List<String> tablesToUpdate = Arrays.asList("routes", "fare_rules", "patterns", "trips");
+        for (String table : tablesToUpdate) {
+            connection.createStatement()
+                .execute(String.format("update %s.%s set route_id = '%s'", testInjectionNamespace, table, injection));
+        }
+        connection.commit();
+
+        // make graphql query
+        Map<String, Object> variables = new HashMap<String, Object>();
+        variables.put("namespace", testInjectionNamespace);
+        assertThat(queryGraphQL("feedRoutes.txt", variables, testInjectionDataSource), matchesSnapshot());
+    }
+
+
+    /**
+     * Helper method to make a query with default variables
+     */
+    private Map<String, Object> queryGraphQL(String queryFilename) throws IOException {
+        Map<String, Object> variables = new HashMap<String, Object>();
+        variables.put("namespace", testNamespace);
+        return queryGraphQL(queryFilename, variables, testDataSource);
+    }
+
+    /**
+     * Helper method to execute a GraphQL query and return the result
+     */
```

Comment:
Please clarify in Javadoc what the parameters do, most importantly queryFilename. It's not obvious in the code that calls this method that the filenames are test fixtures containing the full query text.